anoncreds/protocol/types.py

```diff
@@ -193,6 +193,35 @@ def __eq__(self, other):
                and self.Z == other.Z and self.seqId == other.seqId \
                and dict(self.R) == dict(other.R)
```

Comment:
not yet, because of this code in jsonpickle_util.py
class CommonIntegerElementHandler(jsonpickle.handlers.BaseHandler):
    def flatten(self, obj, data):
        data[DATA_KEY] = obj.toStrDict()
        return data

Revocation part also uses this code